etl/transform/datadiscovery_cards.py

```diff
@@ -268,25 +277,33 @@ def _handle_DbId_URI(document, document_type, documents_dbid_fields_plus_field_t
         document[document_type + 'DbId'] = get_generated_uri_from_dict(source, document_type, document, True)
     # transform other DbIds , skip observationVariable
     if document_type in documents_dbid_fields_plus_field_type:
-        for fields in documents_dbid_fields_plus_field_type[document_type]:
-            if fields[0] in document:
-                if document[fields[0]] and fields[0].endswith("DbIds"):
+        for current_field in documents_dbid_fields_plus_field_type[document_type]:
+            if current_field[0] in document:
+                if document[current_field[0]] and len(current_field)==4 and current_field[2] == "object-list":
```

Comment:
Relying on field size and field type looks quite leak to me, but I don't see really the data structure behind that expectation. An example explaining what it might look would be welcome (at least a link towards an example).

Reply:
Indeed, to keep in a further pull request